src/lib/components/feedback-modal.svelte

```diff
@@ -27,12 +62,25 @@
 		<Typography>{$_('feedback.text')}</Typography>
 		<Textarea placeholder={$_('feedback.textareaPlaceholder')}></Textarea>
 		<section class="share">
-			<Checkbox label={$_('feedback.checkboxLabel')} />
+			<Checkbox label={$_('feedback.checkboxLabel')} bind:checked={isEmailShared} />
 			<Typography variant="small">{$_('feedback.checkboxHelperText')}</Typography>
+			{#if isEmailShared}
+				<Input
+					label="Email"
+					bind:value={email}
+					error={emailTouched && email.trim() !== '' && emailError?.email?._errors
+						? emailErrorSnippet
+						: undefined}
+					onblur={onEmailBlur}
+				/>
+			{/if}
 		</section>
 		<section class="buttons">
-			<Button variant="strong" dimension="compact" onclick={confirm}
-				><SendAlt size={24} />{$_('feedback.confirmButton')}</Button
+			<Button
+				variant="strong"
+				disabled={isEmailShared ? !isEmailValid : false}
```

Comment:
```suggestion
				disabled={isEmailShared && !isEmailValid}
```

It should also validate that the form is not empty.